src/documents/views.py

```diff
@@ -712,3 +714,41 @@ def get(self, request, format=None):
                 "feature_is_set": feature_is_set,
             },
         )
+
+
+class UiSettingsView(GenericAPIView):
+
+    permission_classes = (IsAuthenticated,)
+    serializer_class = UiSettingsViewSerializer
+
+    def get(self, request, format=None):
+        serializer = self.get_serializer(data=request.data)
+        serializer.is_valid(raise_exception=True)
+
+        user = User.objects.get(pk=request.user.id)
+        displayname = user.username
+        if user.first_name or user.last_name:
+            displayname = " ".join([user.first_name, user.last_name])
+        settings = []
```

Comment:
I think this should be `'{}'` ie an empty json looking dict, since that's what the testing seems to expect.  It's probably unlike to happen though, with the auto migration to the DB happening quickly